scality_manila_utils/helper.py

```diff
@@ -156,7 +160,28 @@ def wipe_export(root_export, exports_file, export_name):
     :param export_name: name of export to remove
     :type export_name: string (unicode)
     """
-    raise NotImplementedError
+    export_point = os.path.join('/', export_name)
+    if export_point in _get_defined_exports(exports_file):
+        raise ExportHasGrantsException('Unable to remove export with grants')
+
+    if export_name not in _get_export_points(root_export):
```

Comment:
Should the check of "export existence" be done before the check of "grants" ? It doesn"t matter technically but I expect to "coarse" check to be done first.